docker/entrypoint.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+if [ -n "${HOST_UID}" ]
+  then
+    groupmod --gid=$HOST_GID virtualquake
+    usermod --uid=$HOST_UID virtualquake
+fi
+chown -R virtualquake:virtualquake /home/virtualquake/
+cd /home/virtualquake/external_volume
+su virtualquake
```

Comment:
I've played with a few options for running su and getting the sub shell to move to the right directory, but they have issues like starting multiple nested shells; I'm not really familiar with what problems carrying the root environment to the user could produce, but if it's not critical, the existing `su virtualquake` produces the behavior I want and so I'm good keeping it as is.